src/server/auth/server/api_server.go

```diff
@@ -346,11 +345,10 @@ func (a *apiServer) Activate(ctx context.Context, req *authclient.ActivateReques
 	// Activate() is rarely called, and it helps avoid races due to other pachd
 	// pods being out of date.
 	for {
+		time.Sleep(time.Second) // give other pachd nodes time to update their cache
```

Comment:
Why not just do this with a backoff, it won't really change much but it'll make it look a bit more official. Also it's an easy way to add a notification when we retry (not that it'd be hard to do with this code as is) which I think is a good idea because should this somehow become an infinite loop due to a bug or something we'll at least have a chance at debugging it.

Reply:
Per our conversation, I wanted to preserve the behavior that these methods sleep even after updating their own cache, to give other pachd nodes time to catch up. I changed the implementation, though, so that these now have a backoff followed by a sleep.